recipe/meta.yaml

```diff
@@ -1,44 +1,60 @@
-{% set version = "0.12.0" %}
+{% set name = "folium" %}
+{% set version = "0.14.0" %}
+
 
 package:
-  name: folium
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/f/folium/folium-{{ version }}.tar.gz
-  sha256: d45ace0a813ae65f202ce0356eb29c40a5e8fde071e4d6b5be0a89587ebaeab2
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/folium-{{ version }}.tar.gz
+  sha256: 8ec44697d18f5932e0fdaee8b19da98625de4d0e72cb30ef56f9479f18e11b9f
 
 build:
-  number: 1
-  noarch: python
-  script:
-    - export SETUPTOOLS_SCM_PRETEND_VERSION={{ version }}
-    - {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: True  # [py<35]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps
 
 requirements:
   host:
-    - python >=3.5
+    - python
     - pip
+    - setuptools
+    - setuptools_scm
+    - wheel
   run:
-    - python >=3.5
-    - branca >=0.3.0
+    - python
+    - branca >=0.6.0
     - jinja2 >=2.9
     - numpy
     - requests
 
 test:
+  imports:
+    - folium
   requires:
     - pip
   commands:
     - pip check
-  imports:
-    - folium
 
 about:
   home: https://github.com/python-visualization/folium
```

Comment:
The "official" home page appears to be https://python-visualization.github.io/folium/

Reply:
Thanks. Fixed